pkgs/by-name/in/invidious/package.nix

```
@@ -27,6 +27,7 @@
 let
   # normally video.js is downloaded at build time
   videojs = callPackage ./videojs.nix { inherit versions; };
+  crystal = crystal_1_17;
```

```suggestion
  crystal = crystal_1_16;
```

Version 1.16 remains the recommended version until the following upstream issues are resolved.

- https://github.com/iv-org/invidious/pull/5517
- https://github.com/iv-org/invidious/pull/5338

---

Ah yeah it was using 1.16 all along because of `crystal = crystal_1_16`